CHANGELOG.md

```diff
@@ -185,4 +185,7 @@ v0.6.10
 Add ingestion timestamp filter summary to **NioBuyerStudioSubscription**
 
 0.7.4 
-Fix group-by nested path display
\ No newline at end of file
+Fix group-by nested path display
+
+0.7.5 
```

Comment:
was this a mistake? Or is the latest version going to be 0.7.5 instead of 0.7.6?

Reply:
Good catch, fixed